composer.json

```diff
@@ -25,11 +25,13 @@
     "jms/serializer": "^1.6|^2.0|^3.0",
 
     "goetas-webservices/xsd2php-runtime": "^0.2.11",
-    "goetas-webservices/soap-common": "^0.2.0"
+    "goetas-webservices/soap-common": "dev-master",
```

Comment:
i have released a new version for this https://github.com/goetas-webservices/soap-common/releases/tag/0.2.2

Reply:
Thank you so much!